folder/src/Component/Sliding.tsx

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+import { Accordion, Panel } from "baseui/accordion";
+import { Checkbox, LABEL_PLACEMENT } from "baseui/checkbox";
+import { FormControl } from "baseui/form-control";
+import { Slider } from "baseui/slider";
+
+interface SlidingProps {
+  length: number[];
+  setLength: (value: number[]) => void;
+
+  capitalAllowed: boolean;
+  setCapitalAllowed: (value: boolean) => void;
+
+  numberAllowed: boolean;
+  setNumberAllowed: (value: boolean) => void;
+
+  symbolAllowed: boolean;
+  setSymbolAllowed: (value: boolean) => void;
+}
+
+const Sliding: React.FC<SlidingProps> = ({
+  length,
+  setLength,
+  capitalAllowed,
+  setCapitalAllowed,
+  numberAllowed,
+  setNumberAllowed,
+  symbolAllowed,
+  setSymbolAllowed,
+}) => {
+  return (
+    <Accordion
+      accordion
+      overrides={{
+        Root: {
+          style: {
+            borderRadius: "6px",
+            overflow: "hidden",
+            backgroundColor: "#white",
+          },
```

Comment:
instead of specifying all these styles inline, can you use either CSS/SCSS stylesheets or a separate styles.ts file where all these objects can be put together.

It doesn't look clean to have the HTML + CSS thrown together